purchase_order_report_selection/models/purchase_order.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    report_field_ids = fields.One2many("purchase.report.field", "purchase_order_id")
+
+
+class PurchaseReportField(models.Model):
```

Comment:
maybe will be better to split this model into a separate file e.g. purchase_report_filed.py

Reply:
Got it, I'll do it